user-interface/src/lib/components/uswds/DateRangePicker.tsx

```diff
@@ -103,11 +133,46 @@ function DateRangePickerComponent(props: DateRangePickerProps, ref: React.Ref<Da
         maxDate={maxDate}
         onChange={onEndDateChange}
         label={endDateLabel || 'End date'}
+        ariaDescription={`${id}-aria-description`}
+        ariaLive={ariaLive ?? undefined}
         name="event-date-end"
         value={value?.end}
         disabled={disabled}
         required={required}
       />
+      <span id={`${id}-aria-description`} aria-live="polite" hidden>
+        <span>Format: numeric month / numeric day / 4 digit year.</span>
```

Comment:
How does the screen reader read this? 

Also, I believe the following is generally more grammatically correct:
```suggestion
        <span>Format: numeric month / numeric day / 4-digit year.</span>
```

Reply:
It reads it as "Format" "numeric month slash numeric day slash 4 digit year"

I'll add the hyphen

Reply:
Cool. I was hoping it wasn't doing something weird with the slash.